File: ts-sdk/src/sdk/models/operations/completearevert.ts
import { SpeakeasyBase, SpeakeasyMetadata } from "../../../internal/utils";


export class CompleteARevertPathParams extends SpeakeasyBase {
  @SpeakeasyMetadata({ data: "pathParam, style=simple;explode=false;name=database" })
  database: string;

  @SpeakeasyMetadata({ data: "pathParam, style=simple;explode=false;name=number" })
  number: string;

  @SpeakeasyMetadata({ data: "pathParam, style=simple;explode=false;name=organization" })
  organization: string;
}

export class CompleteARevertRequest extends SpeakeasyBase {
  @SpeakeasyMetadata()
  pathParams: CompleteARevertPathParams;
}

export class CompleteARevertResponse extends SpeakeasyBase {
  @SpeakeasyMetadata()
  contentType: string;

  @SpeakeasyMetadata()
  statusCode: number;

  @SpeakeasyMetadata()
  completeARevert200ApplicationJSONObject?: Record<string, Record<string, any>>;
}

File: go-sdk/pkg/models/operations/completearevert.go
package operations

type CompleteARevertPathParams struct {
	Database     string `pathParam:"style=simple,explode=false,name=database"`
	Number       string `pathParam:"style=simple,explode=false,name=number"`
	Organization string `pathParam:"style=simple,explode=false,name=organization"`
}

type CompleteARevertRequest struct {
	PathParams CompleteARevertPathParams
}

type CompleteARevertResponse struct {
	ContentType                             string
	StatusCode                              int
	CompleteARevert200ApplicationJSONObject map[string]map[string]interface{}
}
